### DISCUSSION TITLE
Welcome to dialoqbase Discussions!

### DISCUSSION THREAD

**Discussion by n4ze3m** [n4ze3m/dialoqbase#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jmanhype** [comment]
HELP Invalid `prisma.user.findFirst()` invocation: The table `public.User` does not exist in the current database.
wont allow me to login I did some trouble shooting and this npx prisma migrate dev --name init

**Reply by n4ze3m** [reply]
I'm sorry to hear that you're facing issues. Did you run it using Docker Compose? If so, are you using Windows?

**Reply by jmanhype** [reply]
yes did docker compose using windows

when that didnt work I tired to put a .env file in the prima folder in order to give it DATABASE_URL="postgres://postgres:postgres@postgres:5432/dialoqbase"  which was stopping me from Database migration hasn't been run yet: Prisma doesn't automatically create tables in your database when you define models in the schema. It requires you to run a migration. If you haven't done that yet, here's how:




STILL NOT WORKING HELPP

**Reply by jmanhype** [reply]
Its working now do you think you could implement flowise?

**Comment by drmatoi** [comment]
Here bevor this get's famous, nice Projekt!

**Reply by n4ze3m** [reply]
😅😅😅😅

**Comment by TommyWong92** [comment]
i always get data source failed on whatever website link i put in, any ideas?

**Reply by n4ze3m** [reply]
For the data source website, I currently use Cheerio. I think I need to use Puppeteer instead. Maybe that's why.

**Reply by n4ze3m** [reply]
If you tried accessing a website blocked by your ISP, it might also fail, I guess, tho

**Reply by TommyWong92** [reply]
i follow your demo in twitter, i pass in website or text and it still fail

**Reply by n4ze3m** [reply]
I think a similar issue was reported about the website's data source. I will check what happened

**Reply by TommyWong92** [reply]
any updates?

**Reply by n4ze3m** [reply]
the issue is caused by redis. I'm trying to migrate it to another queue management system, and it will be finished this weekend or next.

**Reply by TommyWong92** [reply]
![image](https://github.com/n4ze3m/dialoqbase/assets/106639301/e112fd9a-5878-4def-9b4d-ca90bcd3d569)

i doing this in railway, any valid url that i can test the data source?

**Reply by n4ze3m** [reply]
Hey, can you check the railway logs? Also, can we move this conversation to here: https://github.com/n4ze3m/dialoqbase/issues/76
